### DISCUSSION TITLE
colorscheme

### DISCUSSION THREAD

**Discussion by prmadev** [janleigh/dotfiles#33] [Support] [answered]
Hey! Hello! My compliments to the chief!
Are you using a famous colorscheme, or is it custom-made? 
Awesome rice!

**Comment by janleigh** [comment, marked as answer]
> Hey! Hello! My compliments to the chief! Are you using a famous colorscheme, or is it custom-made? Awesome rice!

It is a custom made colorscheme. It currently has no official name yet though.

**Reply by prmadev** [reply]
You should definitely name it and publish it. It is extremely appealing, balanced and accessible!

**Reply by janleigh** [reply]
Only problem is I suck at naming :D

**Reply by Deathemonic** [reply]
how to about what your rice aims to be and how you see it, you can use food, mountains, the wallpaper of the dotfiles or anything as a reference, Like ``Mist Horizon`` or something like that
